notebooks/scraping_x.qmd

```diff
@@ -0,0 +1,206 @@
+---
+jupyter: python3
+---
+
+```{python}
+#| ExecuteTime: {end_time: '2024-10-16T13:55:17.072805Z', start_time: '2024-10-16T13:55:16.661438Z'}
+#| collapsed: true
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.keys import Keys
+import CryptoFraudDetection.scraper.twitter as twitter_scrapper
+from CryptoFraudDetection.utils.logger import Logger
+import time
+
+import json, os
+import csv
+import random
+```
+
+```{python}
+#| ExecuteTime: {end_time: '2024-10-16T13:54:22.195234Z', start_time: '2024-10-16T13:54:22.183531Z'}
+x_scrapper = twitter_scrapper.TwitterScraper(os.getenv("TWITTER_USERNAME"), os.getenv("TWITTER_PASSWORD"), Logger("logger"))
+```
+
+```{python}
+x_scrapper.login_save_cookies()
+```
+
+```{python}
+df_test = x_scrapper.scrape_with_cookies()
+```
+
+```{python}
+df_test['Tweet'].values
+```
+
+```{python}
+df_test
+```
+
+#old / will be deleted if class works
```

Comment:
remove, maybe we dont even need the notebook and create a clean notebook where we scrape all the sources